nixos/modules/services/networking/firewalld/settings.nix

```diff
@@ -0,0 +1,202 @@
+        CleanupModulesOnExit = mkOption {
+          type = bool;
+          description = "Whether to unload all firewall-related kernel modules when firewalld stops.";
+          default = false;
+        };
+        IPv6_rpfilter = mkOption {
+          type = enum [
+            "strict"
+            "loose"
+            "strict-forward"
+            "loose-forward"
+            "no"
+          ];
+          description = ''
+            Performs reverse path filtering (RPF) on IPv6 packets as per RFC 3704.
+
+            Possible values:
+            - strict: Performs "strict" filtering as per RFC 3704. This check
+              verifies that the in ingress interface is the same interface that
+              would be used to send a packet reply to the source. That is,
+              ingress == egress.
+            - loose: Performs "loose" filtering as per RFC 3704. This check only
+              verifies that there is a route back to the source through any
+              interface; even if it's not the same one on which the packet arrived.
+            - strict-forward: This is almost identical to "loose", but does not
+              perform RPF for packets targeted to the host (INPUT).
+            - loose-forward: This is almost identical to "loose", but does not
+              perform RPF for packets targeted to the host (INPUT).
+            - no: RPF is completely disabled.
+
+            The rp_filter for IPv4 is controlled using sysctl.
+          '';
+          default = "strict";
+        };
+        IndividualCalls = mkOption {
+          type = bool;
+          description = ''
+            Whether to use individual -restore calls to apply changes to the
+            firewall. The use of individual calls increases the time that is
+            needed to apply changes and to start the daemon, but is good for
+            debugging as error messages are more specific.
+          '';
+          default = false;
+        };
+        LogDenied = mkOption {
+          type = enum [
+            "all"
+            "unicast"
+            "broadcast"
+            "multicast"
+            "off"
+          ];
+          description = ''
+            Add logging rules right before reject and drop rules in the INPUT,
+            FORWARD and OUTPUT chains for the default rules and also final
+            reject and drop rules in zones for the configured link-layer packet
+            type.
+          '';
+          default = "off";
+        };
+        FirewallBackend = mkOption {
+          type = enum [
+            "nftables"
+            "iptables"
+          ];
+          description = ''
+            The firewall backend implementation. This applies to all firewalld
+            primitives. The only exception is direct and passthrough rules which
+            always use the traditional iptables, ip6tables, and ebtables backends.
+
+            Note: The iptables backend is deprecated. It will be removed in a future release.
```

Comment:
Tip: if you want a "proper" note admonition, you can use:

```markdown
::: {.note}

Text here

:::
```

See https://github.com/NixOS/nixpkgs/blob/master/doc/README.md#admonitions


---

I wonder if this specific call out should be a "warning" or "caution"?